ut/columns_ut.cpp

```diff
@@ -258,3 +259,11 @@ TEST(ColumnsCase, LowCardinalityString_Save) {
         EXPECT_EQ(col.At(i), foobar(i)) << " at pos: " << i;
     }
 }
+
+TEST(ColumnsCase, CreateSimpleAggregateFunction) {
+    auto col = CreateColumnByType("SimpleAggregateFunction(funt, Int32");
```

Comment:
`SimpleAggregateFunction(funt, Int32` (without closing parentheses)? Does this test passes? If yes, then why? Should it pass (i.e. should type parser tolerate this)?